### DISCUSSION TITLE
Ender 6 Klipper Screen Not Working FIX

### DISCUSSION THREAD

**Discussion by S0LV3R** [Desuuuu/DGUS-reloaded-Klipper#44] [General]
If the screen for your Creality Ender 6 is not working check out this article: https://github.com/Desuuuu/DGUS-reloaded-Klipper/wiki/Flashing-the-firmware. The problem is most likely that you didn't download the DWIN_SET.tar.gz file from the right source. You must use the DWIN_SET folder from the latest release archive, not the repository. I downloaded the DWIN_SET.tar.gz from the "latest release" link. I then replaced the T5UID1.CFG file in the DWIN_SET folder with the one found in the article and reflashed the screen firmware. This got my screen working.

**Comment by jjpainkiller** [comment]
Hi there,

I have an issue, the Z buttons on the screen execute inverted movements on the axis, while home button works fine and in web both work fine. Saw a guide somewhere else and didn't see this cfg file tip. Would this fix it? Thanks in advance.

**Reply by gconlon** [reply]
I just did mine and have the same issue - did you find a solution?
